crepecake/src/main/java/com/google/cloud/tools/crepecake/http/Response.java

```diff
@@ -41,6 +42,16 @@ public int getStatusCode() {
     return httpResponse.getHeaders().getHeaderStringValues(headerName);
   }
 
+  /** Gets the first {@code Content-Length} header, or {@code -1} if not found. */
+  public long getContentLength() {
+    String contentLengthHeader =
+        httpResponse.getHeaders().getFirstHeaderStringValue(HttpHeaders.CONTENT_LENGTH);
+    if (contentLengthHeader == null) {
+      return -1;
+    }
+    return Long.parseLong(contentLengthHeader);
```

Comment:
I wonder if we should catch the number format exception thrown by parseLong? In case the response isn't according to spec?

Reply:
or is that simply just not a possibility?

Reply:
Added a check just in case.

Reply:
It's totally a possibility, and we should probably return -1 in this case. Don't bubble the exception here.